ng-dev/utils/bazel-bin.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright Google LLC
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {join} from 'path';
+import {determineRepoBaseDirFromCwd} from './repo-directory.js';
+
+let bazelPath = process.env.BAZEL;
+const bazel = {
+  get BAZEL_BIN() {
```

Comment:
Isn't this getter invoked directly below? 😄 

Love that we respect `BAZEL` env though. I use that to avoid `yarn bazel` in my personal environment

Reply:
I suppose it is, I was hoping to make it lazy, but your write this is just immediate with more steps.